tests/containers/compose.pm

```diff
@@ -72,7 +72,13 @@ sub run {
 
     my $engine = $self->containers_factory($runtime);
 
-    install_packages('docker-compose');
+    my @pkgs = ('docker-compose');
+    # Work-around for https://bugzilla.suse.com/show_bug.cgi?id=1244448
+    # docker-compose package pulls all docker dependencies when used with podman
+    # Note: When `CONTAINER_RUNTIMES=podman,docker` we don't care and it allows
+    # us to test how both runtimes behave when installed together.
+    push @pkgs, 'podman-docker' if check_var("CONTAINER_RUNTIMES", "podman");
```

Comment:
I would recommend to apply this workaround only for SLES16 and mark https://bugzilla.suse.com/show_bug.cgi?id=1244448 as softfailure.

Reply:
> I would recommend to apply this workaround only for SLES16 and mark https://bugzilla.suse.com/show_bug.cgi?id=1244448 as softfailure.

We're only running these tests on Tumbleweed, SLE 16.0 & SLEM 6.x (depending on podman or docker).

I'm against soft-failing it since it's not considered a bug.